.github/workflows/gke-vpa-recommendations-ci.yml

```diff
@@ -15,11 +15,8 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
-      - name: build metric-exporter container
```

Comment:
This container still exist no? Should the "validate terraform" override this step, or create a new one?

Reply:
@bourgeoisor Thank you for catching this. Yes, I still need this container 'metric-exporter'.

removed hpa-metrics-exporter.This container is no longer needed.